src/stellar_service/serializers.py

```
@@ -213,6 +243,65 @@ def create(self, validated_data):
         return admin_account
 
 
+class AdminCreateAssetSerializer(serializers.Serializer):
+    """
+    Create asset issuer account
+    """
+    address = serializers.CharField()
+    currency_code = serializers.CharField()
+
+    def get_user(self):
+        user = None
+        request = self.context.get("request")
+        if request and hasattr(request, "user"):
+            user = request.user
+        return user
+
+    def create(self, validated_data):
+        user = self.get_user()
+        address = validated_data.get('address')
+        currency_code = validated_data.get('currency_code')
+
+        service_account = ServiceAccount.objects.get(company=user['company'])
+
+        hotwallet_account = HotwalletAccount.objects.get(
+            service_account=service_account)
+
+        # Create currency on rehive
+        try:
+            rehive = Rehive(hotwallet_account.service_account.token)
+
+            rehive.admin.currencies.post(
+                code=currency_code,
+                description="token asset",
+                symbol=currency_code,
+                unit=currency_code,
+                divisibility=8,
+                enabled=True
+            )
+            # Add currency to hotwallet account on rehive
+            rehive.admin.accounts.obj(hotwallet_account.rehive_account_reference).currencies.post(
+                currency=currency_code
+            )
+        except Exception as exc:
+            print(exc)
```

This should return a serializer error to the caller. Printing the error here just means it continues through to the next set of logic.